README.md

```diff
@@ -39,35 +51,131 @@ class ListMovingVehicle(vqpy.QueryBase):
     @staticmethod
     def setting() -> vqpy.VObjConstraint:
         filter_cons = {'__class__': lambda x: x == Vehicle,
-                       'bbox_velocity': lambda x: x >= 0.1}
+                       'velocity': lambda x: x >= 0.1}
         select_cons = {'track_id': None,
                        'license_plate': None}
         return vqpy.VObjConstraint(filter_cons=filter_cons,
                                    select_cons=select_cons)
 ```
 
+We also have a utility function named `vqpy.utils.continuing`, which checks whether a condition function continues to be true for a certain duration. The following code demonstrate a query that selects all the `Person` objects whose `bottom_center` property lies in the specified region. Then the `track_id` field, `coordinate` field and `in_roi_periods` field. Note that `in_roi_periods` is generated by the `vqpy.utils.continuing` instance, meaning that the total time period the condition is satisfied.
```

Comment:
Shall we mention that we have some query utility function, for example `continuing`, and give a link to the API doc and the example? Since it is a util function but not a critical concept, maybe we shouldn't include that much detailed explanation in our **get started** doc, which is supposed to help users quickly have an idea about the main concepts as well as the main workflow.  

The detailed explanation should be included in our notebook demo explanations.

Reply:
I think we can simplify this part after having some similar utility functions, because this `vqpy.utils.continuing` improved the expressiveness of VQPy queries.

Reply:
Please resolve this comment, and remove the code snip.